src/prefect/cli/_prompts.py

```diff
@@ -378,8 +378,10 @@ async def prompt_build_custom_docker_image(
             build_step["dockerfile"] = "Dockerfile"
         else:
             if confirm(
-                "A Dockerfile exists but you chose not to use it. Would you like to"
-                " rename it?"
+                "A Dockerfile exists. You chose not to use it."
+                " A temporary Dockerfile will be created and will overwrite"
```

Comment:
Thanks for whipping this up so quick!

One thing: It won't overwrite the existing Dockerfile, it will fail when the build step is run. We should also tell them it will be created during the build step so they don't expect it to be renamed right after this confirmation.

Reply:
Thank you! Updated and added your commit.